packages/core/src/oidc/scope.ts

```diff
@@ -0,0 +1,48 @@
+import { User } from '@logto/schemas';
+import { idTokenClaims, UserClaim, userinfoClaims, UserScope } from '@logto/shared';
+import { Nullable } from '@silverhand/essentials';
+import { ClaimsParameterMember } from 'oidc-provider';
+
+export const claimToUserKey: Readonly<Record<UserClaim, keyof User>> = Object.freeze({
+  name: 'name',
+  picture: 'avatar',
```

Comment:
Since `avatar` is mapped to `picture`, we need to check the usage of `avatar` in our project. E.g., https://github.com/logto-io/logto/blob/e01042cbcd77c486afa1ee9fc2fa5c1d2df92542/packages/console/src/components/AppContent/components/UserInfo/index.tsx#L19

Reply:
good call

Reply:
checked our codebase, only found this file. updated